pmsoService/product_flow/features/deliverytrip/api/view_modules/detail_update_views.py

```
@@ -82,7 +92,10 @@ def patch(self, request, id):
         
         # Convert to command and execute
         status_command = serializer.to_command(str(delivery_trip.id))
-        updated_trip = change_trip_status(status_command, request.user)
+        try:
+            updated_trip = change_trip_status(status_command, request.user)
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)